lib/optimizely.rb

```diff
@@ -267,6 +267,11 @@ def is_feature_enabled(feature_flag_key, user_id, attributes = nil)
         variation = decision['variation']
         # Send event if Decision came from an experiment.
         if decision.source == Optimizely::DecisionService::DECISION_SOURCE_EXPERIMENT
+          unless variation['featureEnabled']
```

Comment:
Property name ‘featureEnabled’ should be declare & used as a String constant not as String literal everywhere.

Reply:
Done